docs/advanced/datasource.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: '💾 Datasource'
+---
+
+## Vector Database
+
+We support `Chromadb` and `Elasticsearch` as two type of vector database. 
+`Chromadb` is used as default.
+
+### App
+```python
+import os
+
+from embedchain import App
+from embedchain.config import AppConfig
+
+os.environ["ES_ENDPOINT"] = "elasticsearch_endpoint"

```

Comment:
Host and Port are database settings that are already part of the `AppConfig`, I know I said it myself, but now I think we should just make this an option of `AppConfig`. Otherwise the user has to import too much boilerplate.

Reply:
moved config to `ElasticsearchDBConfig`